Docs 3386 update dataset articles new (#1613)
truenas/documentation: content/SCALE/SCALEUIReference/Storage/Pools/Datasets/DatasetsScreensScale.md

* **[Roles](#roles-widget)**
* **[ZFS Encryption](#zfs-encryption-widget)**

### Datasets Details Widget
The **Dataset Details** widget lists information on dataset type, and the sync, compression level, case sensitivity, Atime and ZFS deduplication settings. **Path** displays the full path for the selected dataset. 
{{< expand "Click Here for More Information" "v" >}}
A root dataset path displays the pool name alone.

![DatasetDetailsWidgetRootDataset](/images/SCALE/22.12/DatasetDetailsWidgetRootDataset.png "Dataset Details Widget Root Dataset")

A child dataset path displays the root dataset (pool) name and parent dataset.

![DatasetDetailsWidgetChildDataset](/images/SCALE/22.12/DatasetDetailsWidgetChildDataset.png "Dataset Details Widget Child Dataset")

**Edit** opens the **[Edit Dataset](#add-and-edit-dataset-screens)** screen for the selected dataset.

Non-root parent and child dataset versions of the card include the **[Delete](#delete-dataset)** option. 
To delete a root dataset, use the **Disconnect/Export** option on the **[Storage Dashboard]({{< relref "StorageDashboardScreen.md" >}})** screen.
{{< /expand >}}

#### Delete Dataset

> The **Delete** button on the **Dataset Details** widget opens a window that includes information about other options or services that use the dataset, for example a parent to other datasets and the services the child datasets of a parent dataset uses. 
> Non-root parent and child datasets include the **Delete** button. 
> {{< expand "Click Here for More Information" "v" >}}
> The **Delete** window for a parent dataset (non-root) includes information about snapshots, shares or other services such as Kubernetes or VMs that use the dataset.
> If it is a parent to other datasets, the window includes the services a child dataset of this parent dataset uses.

Reviewer: ```suggestion
The **Delete** button on the **Dataset Details** widget opens a window that includes information about other options or services that use the dataset, for example, a parent to other datasets and the services the child datasets of a parent dataset uses. 
Non-root parent and child datasets include the **Delete** button. 
{{< expand "Click Here for More Information" "v" >}}
The **Delete** window for a parent dataset (non-root) includes information about snapshots, shares, or other services such as Kubernetes or VMs that use the dataset.
If it is a parent to other datasets, the window includes the services a child dataset of this parent dataset uses.
```

![DeleteDatasetParentDataset](/images/SCALE/22.12/DeleteDatasetParentDataset.png "Delete Dataset Parent Dataset")

> If a child dataset uses services the window displays them.

Reviewer: ```suggestion
If a child dataset uses services, the window displays them.
```

![DeleteDatasetChildUsingAService](/images/SCALE/22.12/DeleteDatasetChildUsingAService.png "Delete Dataset Child Dataset Using a Service")

If a child dataset is not used by a service, it does not display a service.

![DeleteDatasetChildDataset](/images/SCALE/22.12/DeleteDatasetChildDataset.png "Delete Dataset Child Dataset")

> The window includes field where you type the path for the dataset and a **Confirm** option you must select to activate the **Delete Dataset** button.

Reviewer: ```suggestion
The window includes a field where you type the path for the dataset and a **Confirm** option you must select to activate the **Delete Dataset** button.
```

{{< /expand >}}

### Dataset Space Management Widget
The **Dataset Space Management** widget displays space allocation (reserved, used, available) for all datasets.  
{{< expand "Click Here for More Information" "v" >}}
The widget displays after unlocking encrypted datasets. 
The widget donut graph provides at-a-glance information and numeric values for the space allocated and used in the selected dataset. 
This includes data written and space allocated to child datasets of this dataset. 
It provides access to quota configuration options for the parent dataset and the child dataset of the parent, and for users and groups with access to the dataset.

![DatasetSpaceManagementWidgetRootDataset](/images/SCALE/22.12/DatasetSpaceManagementWidgetRootDataset.png "Dataset Space Management Widget Root Dataset")

**Edit** opens the **[Capacity Settings]({{< relref "CapacitySettingsSCALE.md" >}})** screen where you can set quotas for the dataset.

**[Manage User Quotas]({{< relref "ManageQuotas.md" >}})** screen and **Manage Group Quotas** opens the **[Manage Group Quotas]({{< relref "ManageQuotas.md" >}})** screen.
{{< /expand >}}

### Data Protection Widget

> The **Data Protection** widget displays for all datasets. 
> This widget provides information on the number snapshots and other data protection related scheduled tasks (replication, cloud sync, rsync and snapshots) configured on the system. 

Reviewer: ```suggestion
The **Data Protection** widget displays for all datasets. 
This widget provides information on the number of snapshots and other data protection-related scheduled tasks (replication, cloud sync, rsync, and snapshots) configured on the system. 
```

**Manage Cloud Sync Tasks** opens the **Data Protection > [Cloud Sync Tasks]({{< relref "CloudSyncTasksScreensSCALE.md" >}})** screen list view where you can manage scheduled cloud sync tasks.

**Manage Rsync Tasks** opens the **Data Protection > [Rsync Tasks]({{< relref "RsyncTasksScreensSCALE.md" >}})** screen list view where you can manage scheduled rsync tasks.

{{< /expand >}}

### Permissions Widget
The **Permissions** widget displays for all datasets. 
It indicates the type of ACL as either NFSv4 or Unix Permissions (POSIX) and lists access control items and the owner and group for the dataset.
{{< expand "Click Here for More Information" "v" >}}
Root dataset permissions are not editable. 
Permission screen and widget options vary based on the ACL type. 

![PermissionsWidgetRootDataset](/images/SCALE/22.12/PermissionsWidgetRootDataset.png "Permissions Widget Root Dataset")

Parent and child dataset permissions are editable. 

![PermissionsWidgetParentDataset](/images/SCALE/22.12/PermissionsWidgetParentDataset.png "Permissions Widget Parent or Child Dataset")

> If the ACL type is NFSv4 (the default ACL type) the widget turns the items listed on the **Permissions** widget into buttons that open a configuration are where you can edit the item from the **Permissions** widget. 
> The expanded item configuration area has both **Permissions Advanced** and **Flags Advanced** check-buttons you can select or deselect common NFSv4 permission options for each item type.

Reviewer: ```suggestion
If the ACL type is NFSv4 (the default ACL type) the widget turns the items listed on the **Permissions** widget into buttons that open a configuration where you can edit the item from the **Permissions** widget. 
The expanded item configuration area has both **Permissions Advanced** and **Flags Advanced** check buttons. You can select or deselect common NFSv4 permission options for each item type.
```

|------|---------------|-------------|
| System dataset | [Manage Advanced Settings]({{< relref "AdvancedSettingsScreen.md" >}}) | Select the option to configure the **System Dataset** |
| Apps | [Manage Apps Settings]({{< relref "AppsScreensSCALE.md" >}}) | Displays Kubernetes is using the dataset. Select the option to **Choose Pool** from the **Settings** dropdown list on the **Applications** screen. |
| SMB share | [Manage SMB Shares]({{< relref "SMBSharesScreens.md" >}}) | Displays the name of the SMB share using the dataset. Select it on the **SMB Shares** screen to edit it. |
| Other share | Link to the Share type screen | Displays the name of the share using the dataset. Select it on the share screen (NFS, iSCSI or WebDAV) to edit it. |
| VM | [Manage VM Settings]({{< relref "VirtualizationScreens.md" >}}) | Displays the name of the VM using the dataset(zvol). Select it on the **Virtual Machines** screen to edit it. |

{{< /expand >}}

### ZFS Encryption Widget

The **ZFS Encryption** widget displays for root, non-root parent, and child datasets configured with encryption but the options in the widget vary based on the type of dataset. 

> It includes the current state of the dataset encryption, the encryption root, type andalgorithm used.

Reviewer: ```suggestion
It includes the current state of the dataset encryption, the encryption root, the type, and the algorithm used.
```

{{< expand "Click Here for More Information" "v" >}}

> The **ZFS Encryption** widget displays the **Lock** or **Unlock** options are not available on the root dataset or a child dataset of a non-root parent it inherits encryption settings from. 

Reviewer: ```suggestion
The **ZFS Encryption** widget displays the **Lock** or **Unlock** options. They are not available on the root dataset or a child dataset of a non-root parent it inherits encryption settings from. 
```

![ZFSEncryptionWidgetRootDataset](/images/SCALE/22.12/ZFSEncryptionWidgetRootDataset.png "ZFS Encryption Widget Root Dataset")

Parent or child dataset **ZFS Encryption** widgets include the options to **Lock** and **Unlock** the dataset and to **Edit** the encryption settings.

![ZFSEncryptionWidgetChildDatasetUnlocked](/images/SCALE/22.12/ZFSEncryptionWidgetChildDatasetUnlocked.png "ZFS Encryption Widget Child Dataset Unlocked")

Child dataset **ZFS Encryption** widgets include the **Go to Encryption Root** when you select **Inherit** as its **Encryption Options** setting. The non-root parent dataset controls the state of the child dataset.

![ZFSEncryptionWidgetWithGoToEncryptionRoot](/images/SCALE/22.12/ZFSEncryptionWidgetWithGoToEncryptionRoot.png "ZFS Encryption Widget with Go To Encryption Root")

**Edit** opens the **[Edit Encryption Options]({{< relref "EncryptionUISCALE.md" >}}) for *dataset*** window for the selected dataset.

For more details on encryption windows and functions see [Encryption Settings]({{< relref "EncryptionUISCALE.md" >}}).
{{< /expand >}}

## Add and Edit Dataset Screens

> The **Add Dataset** and **Edit Dataset** screens include the same settings but you cannot change the dataset **Name**, **Share Type** or **Case Sensitivity** settings after you click **Save** on the **Add Dataset** screen. 
> After adding a dataset, to edit encryption options use the **Edit** button on the **ZFS Encryption** widget.
> 
> There are two screen options, **Basic Options** and **Advanced Options**. 
> The **Advanced Options** screen include all the settings found on the **Basic Options** screen.

Reviewer: ```suggestion
The **Add Dataset** and **Edit Dataset** screens include the same settings but you cannot change the dataset **Name**, **Share Type**, or **Case Sensitivity** settings after you click **Save** on the **Add Dataset** screen. 
After adding a dataset, to edit encryption options use the **Edit** button on the **ZFS Encryption** widget.

There are two screen options, **Basic Options** and **Advanced Options**. 
The **Advanced Options** screen includes all the settings found on the **Basic Options** screen.
```

The **Basic Options** include three sections: **[Name and Options](#name-and-options-settings)**, **[Encryption Options](#encryption-options-settings)** and **[Other Options](#other-option-settings---advanced-options)**.

The **Advanced Options** settings include quotas management tools for **This Dataset** and **This Dataset and Child Datasets**, and includes more **Other Options** settings not available on the **Basic Options** screen.

### Name and Options Settings

> These settings are common to both the **Basic Options** and **Advanced Options** screens. 
> Setting include name, path and other general settings.

Reviewer: ```suggestion
These settings are common to both the **Basic Options** and **Advanced Options** screens. 
Settings include name, path, and other general settings.
```
